src/qilib/configuration_helper/adapters/ami430_instrument_adapter.py

```diff
@@ -40,7 +41,9 @@ def __init__(self, address: str, instrument_name: Optional[str] = None) -> None:
         super().__init__(address)
         ip_and_port = address.split(':')
```

Comment:
I do know that you did not touch this code. But if the other comments will lead to a new commit, could you change this to `ip, port = address.split(':')`?

This will lead to a code change in the instantiation of the instrument.

Reply:
sure 👍

Reply:
you want me to do this mod in another commit?

Reply:
Meh, the other comments were resolved, so lets leave this for another commit. It is just about boyscouting